decko/lib/decko/tasks/decko.rake

```diff
@@ -94,6 +94,7 @@ decko_namespace = namespace :decko do
       decko_namespace["migrate"].invoke
       decko_namespace["reset_tmp"].invoke
       Card::Cache.reset_all
+      decko_namespace["mod_install"].invoke
```

Comment:
always run `rake decko:mod_install` on `decko update`.  Incentive to optimize...